lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -523,7 +561,9 @@ LowererMDArch::LowerCallArgs(IR::Instr *callInstr, ushort callFlags, Js::ArgSlot
         callInstr->InsertBefore(argInstr);
         argCount++;
     }
-
+#ifndef _WIN32
+    this->xplatCallArgs.StopRecording();
```

Comment:
No, we only record *register args*, which is what you currently doing inside `GetArgSlotOpnd`. You are not recording stack args.

Recording all might be more clear, otherwise you have to assume this is the only place xmm args could be loaded, all others in int registers, other registers may have no recordings, making the homing code complicated... (If recording all is hard to do right, this is fine with me)